doc/Makefile.am

```diff
@@ -99,14 +99,44 @@ doxygen_sources = \
 	$(top_srcdir)/tagmanager/src/tm_source_file.[ch] \
 	$(top_srcdir)/tagmanager/src/tm_workspace.[ch]
 
-Doxyfile.stamp: Doxyfile $(doxygen_sources)
-	$(AM_V_GEN)$(DOXYGEN) Doxyfile && echo "" > $@
+# set WARN_IF_UNDOCUMENTED because apparently doxygens warns for undocumented stuff
+# in headers (even though it's correctly documented in the corresponding .c file) only
+# for xml output
+Doxyfile-gi: Doxyfile
+	$(AM_V_GEN)$(SED) \
+		-e 's,addtogir=@internal,addtogir=,' \
+		-e 's,^\(GENERATE_HTML.*\)YES,\1NO,' \
+		-e 's,^\(GENERATE_XML.*\)NO,\1YES,' \
+		-e 's,^\(WARN_IF_UNDOCUMENTED.*\)YES,\1NO,' \
+		-e 's,^\(SORT_MEMBER_DOCS.*\)YES,\1NO,' \
+		-e 's,^\(SORT_BRIEF_DOCS.*\)YES,\1NO,' \
+		$< > $@ || { $(RM) $@ && exit 1; }
 
-all-local: Doxyfile.stamp
+Doxyfile.stamp: Doxyfile Doxyfile-gi $(doxygen_sources)
+	$(AM_V_GEN)$(DOXYGEN) Doxyfile-gi && $(DOXYGEN) Doxyfile && echo "" > $@
```

Comment:
Am 11. Februar 2016 18:38:18 MEZ, schrieb Colomban Wendling <notifications@github.com>:
>>  
>> -all-local: Doxyfile.stamp
>> +Doxyfile.stamp: Doxyfile Doxyfile-gi $(doxygen_sources)
>> +	$(AM_V_GEN)$(DOXYGEN) Doxyfile-gi && $(DOXYGEN) Doxyfile && echo ""
>> $@
>
>BTW, will Doxygen reuse some of its findings for the second run?  I
>know you'll hate me because we wanted separate doxyfiles not to alter
>the existing doc, but as Docygen is relatively slow, if it could reuse
>its database or something it'd be super nice :)
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/geany/geany/pull/890/files#r52637124

No, you can't have both. This was one of the reasons for me preferring a single doxyfile